birdhouse/optional-components/README.rst

```diff
@@ -0,0 +1,140 @@
+Optional components
```

Comment:
Not deleting the old optional-components/README.md ?

Reply:
This has now been addressed.